NativeUI/Items/UIMenuSliderItem.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System.Collections.Generic;
+using System.Drawing;
+
+namespace NativeUI
+{
+    public class UIMenuSliderItem : UIMenuItem
+    {
+        protected Sprite _arrowLeft;
+        protected Sprite _arrowRight;
+
+        protected UIResRectangle _rectangleBackground;
+        protected UIResRectangle _rectangleSlider;
+        protected UIResRectangle _rectangleDivider;
+        
+        protected int _value = 0;
+        protected int _max = 100;
+        protected int _multiplier = 5;
+
+
+        /// <summary>
+        /// Triggered when the slider is changed.
+        /// </summary>
+        public event ItemSliderEvent OnSliderChanged;
+
+
+        /// <summary>
+        /// The maximum value of the slider.
+        /// </summary>
+        public int Maximum
+        {
+            get
+            {
+                return _max;
+            }
+            set
+            {
+                if (_value > value)
+                {
+                    _value = value;
+                }
```

Comment:
Should set _max = value;

Reply:
I forgot about that. Fixed on 23b7dec150308f996249f81eda04fcdd45292f65.